pulsar-broker/src/test/java/org/apache/pulsar/client/api/SimpleProducerConsumerTest.java

```diff
@@ -2333,4 +2448,40 @@ public EncryptionKeyInfo getPrivateKey(String keyName, Map<String, String> keyMe
         log.info("-- Exiting {} test --", methodName);
     }
 
+    @Test
+    public void testConsumerSubscriptionInitialize() throws Exception {
```

Comment:
how about name it to testConsumerSubscriptionInitializePosition?

Reply:
Not sure why a lot of un-related changes are bring in.  They should be already in master branch. We may only need to see this method changes in this file?